lib/ansible/modules/remote_management/redfish/idrac_redfish_config.py

```diff
@@ -184,8 +210,10 @@ def main():
     timeout = module.params['timeout']
 
     # iDRAC attributes to update
-    mgr_attributes = {'mgr_attr_name': module.params['manager_attribute_name'],
-                      'mgr_attr_value': module.params['manager_attribute_value']}
+    mgr_attributes = {
```

Comment:
Should this only be initialized this way when manager_attribute_name in present, else initialized to {}?

Reply:
fixed